amavisvt/client.py

```diff
@@ -489,6 +489,51 @@ def is_included(resource):
                     lambda r: '.' not in r.filename or r.filename.endswith('.')
         ]))
 
+    def api_call_check(self, checksums):
+       for api_key in self.config.apikeys:  
+           use_api_key = self.memcached.get(api_key)
+           if not use_api_key:
+               response = requests.post(self.config.api_url, {
+                'apikey': api_key,
+                              'resource': ', ' . join(checksums)
+                              }, timeout=float(self.config.timeout), headers={
+                              'User-Agent': 'amavisvt/%s (+https://ercpe.de/projects/amavisvt)' % VERSION
+                              })
+               response.raise_for_status()
+               if response.status_code == 204:
```

Comment:
I wonder if it is safe to hard code 204 here. I would expect that every api call counts against the API limit, no matter what the response code is.